src/crate/theme/rtd/crate/sidebartoc.html

```diff
@@ -144,10 +144,11 @@
   {% endif %}
 
   <!-- Section D. -->
-  <li class="navleft-item border-top"><a href="/support/">Support</a></li>
-  <li class="navleft-item"><a href="https://community.crate.io/">Community</a></li>
-  <li class="navleft-item"><a href="https://community.crate.io/t/overview-of-cratedb-integration-tutorials/1015">Integration Tutorials</a></li>
-  <li class="navleft-item"><a href="https://github.com/crate/crate-sample-apps">Sample Applications</a></li>
+  <li class="navleft-item border-top"><a target="_blank" href="/support/">Support</a></li>
+  <li class="navleft-item"><a target="_blank" href="https://learn.cratedb.com">Academy</a></li>
```

Comment:
I'd suggest moving this to the bottom of the list. Support and Community should be next to each other. Academy is education and belongs with Tutorials/sample apps.